packages/mobile/src/components/offline-downloads/DownloadToggle.tsx

```diff
@@ -186,7 +186,7 @@ export const DownloadToggle = ({
         <Switch
           value={isCollectionMarkedForDownload}
           onValueChange={handleToggleDownload}
-          disabled={!collectionId || disabled}
```

Comment:
Was there a reason we were checking for collectionId previously?

Reply:
Yeah, but I added the isFavoritesDownload flag since then, and the check up above should cover this. the check we would use here is `(!collectionId && !isFavoritesDownload) || disabled`

Reply:
Yeah, but I added the isFavoritesDownload flag since then, and the check up above should cover this. the check we would use here is `(!collectionId && !isFavoritesDownload) || disabled`

Reply:
cool sounds good